src/classes/OPP_OpportunityAmountSync_SVC.cls-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>44.0</apiVersion>
```

Comment:
Should be 45.0 at this point, I believe.

Reply:
You can go up to 46 now too. We're advising that all the new BDI classes be set at 46 as well.